gauge-card/gauge-card.js

```diff
@@ -151,9 +151,17 @@ class GaugeCard extends HTMLElement {
     return severityMap["normal"];
   }
 
+  _getEntityStateValue(entity, attribute) {
+    if (!attribute) {
```

Comment:
return attribute?entity.attributes[attribute]:entity.state;

Reply:
I tend to avoid the ternary operator because of its bad readability... I'll change it if you insist, but honestly I'd prefer to keep it as is.

Reply:
I don't have a preference, if it helps maintainability, happy to keep it as it is